shard/src/main/java/com/redhat/service/bridge/shard/OperatorServiceInMemoryImpl.java

```diff
@@ -1,87 +1,52 @@
 package com.redhat.service.bridge.shard;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.redhat.service.bridge.infra.dto.BridgeDTO;
-import com.redhat.service.bridge.infra.dto.BridgeStatus;
 import com.redhat.service.bridge.infra.dto.ProcessorDTO;
-import com.redhat.service.bridge.ingress.IngressService;
-import com.redhat.service.bridge.shard.controllers.ProcessorController;
-
-import io.quarkus.scheduler.Scheduled;
+import com.redhat.service.bridge.infra.k8s.K8SBridgeConstants;
+import com.redhat.service.bridge.infra.k8s.KubernetesClient;
+import com.redhat.service.bridge.infra.k8s.crds.BridgeCustomResource;
+import com.redhat.service.bridge.infra.k8s.crds.ProcessorCustomResource;
 
 @ApplicationScoped
 public class OperatorServiceInMemoryImpl implements OperatorService {
```

Comment:
Do we really need to preserve this class? It looks now like a wrapper of `KubernetesClient` so we can probably remove it